src/winter_onboarding_2021/fleet_management_service/core.clj

```diff
@@ -0,0 +1,29 @@
+(ns winter-onboarding-2021.fleet-management-service.core
+  (:require [ring.adapter.jetty :as raj]
+            [winter-onboarding-2021.fleet-management-service.routes :as routes])
+  (:gen-class))
+
+(defonce server (atom nil))
```

Comment:
Instead of using an atom here, we can use the mount library (or some other alternative like integrant, component). 

The handle state similarly, but also allow you to create dependency graphs. For example, in the future, to start a server you'd need a database connection, read the env file, etc.

Reply:
Done